docs/openapi.yaml

```diff
@@ -0,0 +1,399 @@
+        '401':
+          description: Unauthorized, may be caused by expired token.
+
+  /signals/v1/public/terms/categories:
+    get:
+      description: Categories known to SIA.
+      responses:
+        '200':
+          description: List of all main categories with embedded sub categories.
+
+  # This seems not to work, while the generated URI is correct and data is
+  # returned from API. Checked via web inspector.
+  /signals/v1/public/terms/categories/{main_slug}:
+    get:
+      description: Information about a specific main category and its sub categories.
+      response:
+        '200':
+          description: Detailed information.
+      parameters:
+        - name: main_slug
+          in: path
+          description: Slug that identifies the main category.
+          required: true
+          schema:
+            type: string
+
+  # This seems not to work, while the generated URI is correct and data is
+  # returned from API. Checked via web inspector.  
+  /signals/v1/public/terms/categories/{main_slug}/sub_categories/{sub_slug}:
+    get:
+      description: Information about a specific sub category.
+      responses:
+        '200':
+          description: Detailed information.
+      parameters:
+        - name: main_slug
+          in: path
+          description: Slug that identifies the main category.
+          required: true
+          schema:
+            type: string
+        - name: sub_slug
+          in: path
+          description: Slug that identifies the sub category.
+          required: true
+          schema:
+            type: string
+
+  /signals/user/auth/me:
+    get:
+      description: Information about the currently authenticated user.
+      responses:
+        '200':
+          description: User information including permissions.
+        '401':
+          description: Unauthorized, may be caused by expired token.
+      security:
+      - siaAuth:
+        - SIG/ALL
+
+components:
+  schemas:
+
+    signalCreate:
+      type: object
+      properties:
+        source:
+          type: string
+        text:
+          type: string
+          description: The complaint text.
+        text_extra:
+          type: string
+          nullable: true
+        location:
+          $ref: '#/components/schemas/locationInput'
+        category:
+          $ref: '#/components/schemas/categoryInput'
+        reporter:
+          $ref: '#/components/schemas/reporterInput'
+        priority:
+          $ref: '#/components/schemas/priorityInput'
+        notes:
+          $ref: '#/components/schemas/notesInput'
+        incident_date_start:
+          type: string
+          format: date-time
+        incident_date_end:
+          type: string
+          format: date-time
+          nullable: true
+        operational_date:
+          type: string
+          format: date-time
+          nullable: true
+        extra_properties:
+          type: object
+          additionalProperties: yes
```

Comment:
'yes' doesn't validate. Should be object or boolean. ;)

Reply:
done